snippets/mssql.json

```diff
@@ -294,5 +294,54 @@
 			"GO"
 		],
 		"description": "Get Space Used by Tables"
+	},
+
+	"Lists all procedures by schema": {
+		"prefix": "sqlListProcedures",
+		"body": [
+			"-- Lists all procedures by schema ${1:ShemaName}",
+            "SELECT",
+            "SchemaName = s.name,",
+            "ProcedureName = pr.name,",
+            "ProcId = pr.object_id",
+            "FROM",
+            "sys.procedures pr",
+            "INNER JOIN",
+            "sys.schemas s ON pr.schema_id = s.schema_id",
+            "WHERE",
+            "s.name = '${1:ShemaName}'",
+			"GO"
+		],
+		"description": "Lists all procedures by schema"
+	},
+
+
+	"Makes Header Documentation for SQL Stored Procedure": {
+		"prefix": "sqlDocProcedure",
+		"body": [
+            "--",
+            "-- STORED PROCEDURE",
+            "--     ${1:ProcedureName}",
+            "--",
+            "-- DESCRIPTION",
+            "--     ${2:BusinessProcedureDescription}",
+            "--",
+            "-- PARAMETERS",
+            "--     @${3:ParameterName}",
```

Comment:
These are a lot of input values to be required. I would suggest using tab targets for most of these instead - just use `$3` to add a tab target instead of `${3:MyValueName}`. Ideally you should have maybe 3, max 4 things that need to be filled in here, and the rest are just locations you can tab to and add optional information.